packages/utilities/etc/utilities.api.md

```diff
@@ -1,1234 +1,1234 @@
-
-// @public
-export function styled<TComponentProps extends IPropsWithStyles<TStyleProps, TStyleSet>, TStyleProps, TStyleSet extends IStyleSet<TStyleSet>>(Component: React.ComponentClass<TComponentProps> | React.FunctionComponent<TComponentProps>, baseStyles: IStyleFunctionOrObject<TStyleProps, TStyleSet>, getProps?: (props: TComponentProps) => Partial<TComponentProps>, customizable?: ICustomizableProps, pure?: boolean): React.FunctionComponent<TComponentProps>;
-
-// @public (undocumented)
-export function styled<TComponentProps extends IPropsWithStyles<TStyleProps, TStyleSet> & React.RefAttributes<TRef>, TStyleProps, TStyleSet extends IStyleSet<TStyleSet>, TRef = unknown>(Component: React.ComponentClass<TComponentProps> | React.FunctionComponent<TComponentProps>, baseStyles: IStyleFunctionOrObject<TStyleProps, TStyleSet>, getProps?: (props: TComponentProps) => Partial<TComponentProps>, customizable?: ICustomizableProps, pure?: boolean): React.ForwardRefExoticComponent<React.PropsWithoutRef<TComponentProps> & React.RefAttributes<TRef>>;
-
-// @public (undocumented)
-export type StyleFunction<TStyleProps, TStyleSet> = IStyleFunctionOrObject<TStyleProps, TStyleSet> & {
-    __cachedInputs__: (IStyleFunctionOrObject<TStyleProps, TStyleSet> | undefined)[];
-    __noStyleOverride__: boolean;
-};
-
-// @public
-export const tableProperties: Record<string, number>;
-
-// @public
-export const tdProperties: Record<string, number>;
-
-// @public
-export const textAreaProperties: Record<string, number>;
-
-// @public
-export const thProperties: Record<string, number>;
-
-// @public
-export function toMatrix<T>(items: T[], columnCount: number): T[][];
-
-// @public
-export const trProperties: Record<string, number>;
-
-// @public
-export function unhoistMethods(source: any, methodNames: string[]): void;
-
-// @public
-export function useCustomizationSettings(properties: string[], scopeName?: string, localSettings?: ICustomizations): ISettings;
-
-// @public
-export function useFocusRects(rootRef?: React.RefObject<HTMLElement>): void;
-
-// @public
-export function values<T>(obj: any): T[];
-
-// @public
-export const videoProperties: Record<string, number>;
-
-// @public
-export function warn(message: string): void;
-
-// @public
-export function warnConditionallyRequiredProps<P>(componentName: string, props: P, requiredProps: string[], conditionalPropName: string, condition: boolean): void;
-
-// @public
-export function warnControlledUsage<P>(params: IWarnControlledUsageParams<P>): void;
-
-// @public
-export function warnDeprecations<P>(componentName: string, props: P, deprecationMap: ISettingsMap<P>): void;
-
-// @public
-export function warnMutuallyExclusive<P>(componentName: string, props: P, exclusiveMap: ISettingsMap<P>): void;
-
-
-// (No @packageDocumentation comment for this package)
-
-```
+## API Report File for "@uifabric/utilities"
```

Comment:
not sure why this entire file is different,... whitespace? line endings?

Reply:
Probably. It took over 5 mins for all 31 commit hooks to finish a merge conflict resolution.

Reply:
weird! not sure.